client/src/main/java/io/pravega/client/segment/impl/AsyncSegmentInputStream.java

```diff
@@ -35,4 +35,7 @@
 
     @Override
     public abstract void close();
+    
+    public abstract boolean isClosed();
+    
```

Comment:
sorry for the confusion, the new line should be added after `}`  and not before. 
I guess that is the end of file.
it should be 
```
     @Override
      public abstract void close();
+    
+    public abstract boolean isClosed();
}
+
```